src/objects/Course_Offering_Period__c.object

```diff
@@ -0,0 +1,193 @@
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Saturday__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>The day on which the course is offered (Monday - Sunday).</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The day on which the course is offered (Monday - Sunday).</inlineHelpText>
+        <label>Saturday</label>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Sunday__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>The day on which the course is offered (Monday - Sunday).</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The day on which the course is offered (Monday - Sunday).</inlineHelpText>
+        <label>Sunday</label>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Thursday__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>The day on which the course is offered (Monday - Sunday).</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The day on which the course is offered (Monday - Sunday).</inlineHelpText>
+        <label>Thursday</label>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Tuesday__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>The day on which the course is offered (Monday - Sunday).</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The day on which the course is offered (Monday - Sunday).</inlineHelpText>
+        <label>Tuesday</label>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Wednesday__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>The day on which the course is offered (Monday - Sunday).</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The day on which the course is offered (Monday - Sunday).</inlineHelpText>
+        <label>Wednesday</label>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <label>Course Offering Period</label>
+    <listViews>
+        <fullName>All</fullName>
+        <filterScope>Everything</filterScope>
+        <label>All</label>
+    </listViews>
+    <nameField>
+        <label>Course Offering Period Name</label>
+        <trackHistory>false</trackHistory>
+        <type>Text</type>
```

Comment:
@xinnanlisf & @robertbwatson -- question for you both. I'm leaning towards auto-numbering the Course Offering Schedule name rather than having a user type something in there. thoughts?

Reply:
Sounds good to me. I honestly thought it would have been, I must have just missed this.

Reply:
Makes sense to me too!